packages/blade/src/components/Table/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './TableFooter';
 export * from './TablePagination';
 export * from './TableToolbar';
 export * from './types';
+export { TableEditableCell, TableEditableDropdownCell } from './TableEditableCell';
```

Comment:
We should do named exports for the above exports too. I think we may be exposing a lot of unneeded internal components too like CellWrapper and StyledCell